atlp-rwanda/commanders-medica-backoffice
# Feature: Profile settings

**What does this PR do?**

This PR includes designs and implementations of the profile settings page.

**Description of Task to be completed?**
 
- [x]  Design settings page
- [x]   Fetch data from the doctor's table
- [x]   Updating profile
- [x]  Retrieving reviews of the doctor
- [x]  Resetting password

**How should this be manually tested?**
Start the app with **npm run dev** and open the link. Signup then click on settings on the dashboard. On the settings page, you can see reviews, edit your profile, and change your password

Any background context you want to provide?
First signup on this PR the existing account that was not created from this PR updating the profile won't work.
What are the relevant pivotal tracker/Trello stories?
N/A

Screenshots (if appropriate)
N/A

Questions:
N/A
